hircluster.c

```diff
@@ -2008,42 +2007,29 @@ int redisClusterConnect2(redisClusterContext *cc) {
     return _redisClusterConnect2(cc);
 }
 
+/* Get a viable connection context to a given cluster node.
+ * Will connect to the node, setup TLS and authenticate when required.
+ * If an existing context exist but with errors it will be re-created.
+ */
```

Comment:
So this function never returns a connection in an error state? It aways returns NULL in that case?

Reply:
Yes, the idea was to make sure its always returns a connection context without any known errors.
NULL if a connect/reconnect attempt fails

Reply:
OK, good. I think the comment can make that even more clear.